drivers/adc/ad7616/ad7616.c

```diff
@@ -595,8 +1041,15 @@ int32_t ad7616_setup(struct ad7616_dev **device,
 
 	*device = dev;
 
-	if (!ret)
-		printf("AD7616 successfully initialized\n");
```

Comment:
It's because the UART interface is not enabled at this point for stm32 platform (it's initialized in iio_app). I could enable UART before in the new stm32 project #2000 but I feel like a print do not belong in a driver and it's redundant with the return value.

Let me know what you think